Adds a new lowpop (Crash) map, Desparity.

chizzy376 opened the pull request:
## About The Pull Request
Good morning Marines! We have received a distress signal from a freshly established colony similar to LV624! Unfortunate thing is, you are the source of the signal, good luck surviving in this harsh environment!
![map](https://user-images.githubusercontent.com/80391698/227720301-4b178b7e-0b5d-45cb-a4ce-db8329296c57.png)


**The map uses weedable jungle grass, so you won't get the LV624 jungle experience**




--
## Why It's Good For The Game
More map variety good, need to get this out here so people can properly critique the map after playing it. Much better to get player feedback rather than speculation.
### Safe to merge
## Changelog
:cl:
add: Added a new map, Desparity. Lowpop exclusive (30 pop or less).
/:cl:

Losenis commented:
>All that green

That better be weedable

chizzy376 commented:
> > All that green
> 
> That better be weedable

It is

Lumipharon commented on code/game/turfs/floor_ground.dm:
```diff
@@ -109,6 +110,8 @@
 	smoothing_groups = list(SMOOTH_GROUP_JUNGLE_FLOOR)
 
+/turf/open/ground/grass/weedable
+	name = "soft grass"
```
```suggestion
	name = "soft grass"
	
```

chizzy376 replied:
Done

UntoldTactics commented:
Decent map really, though I think it needs to be cut down in regards to large open areas. Maps designed for crash should not have that much area inherently advantageous to marines on the get-go, at least, not so close to disks. Only takes a single flamer to reduce any maze that could be made in crash into nothing.

chizzy376 commented:
> Decent map really, though I think it needs to be cut down in regards to large open areas. Maps designed for crash should not have that much area inherently advantageous to marines on the get-go, at least, not so close to disks. Only takes a single flamer to reduce any maze that could be made in crash into nothing.

Which areas specifically? I'll assume the spot south of engi, east of green? I could add a building there in place of the wide sand path.

UntoldTactics commented:
> Which areas specifically? I'll assume the spot south of engi, east of green? I could add a building there in place of the wide sand path.

Pretty much. Crash is a pretty unmaintained mode, there's a specific 'goldilocks zone' where xenos have lower pop than marine numbers, which results in open areas essentially becoming death zones. I haven't played the map enough to highlight spots, though just wanted to bring up awareness that open areas are a significant advantage towards marines and disadvantage to xenos.

chizzy376 commented:
> > Which areas specifically? I'll assume the spot south of engi, east of green? I could add a building there in place of the wide sand path.
> 
> Pretty much. Crash is a pretty unmaintained mode, there's a specific 'goldilocks zone' where xenos have lower pop than marine numbers, which results in open areas essentially becoming death zones. I haven't played the map enough to highlight spots, though just wanted to bring up awareness that open areas are a significant advantage towards marines and disadvantage to xenos.

Updated the map, updated the pictures as well

chizzy376 commented:
Someone fix this shit yourself, I don't get paid to slave over some moron's "CI Suite / Compile Maps (pull_request) Failing after 15s", who the fuck made this shit, I didn't change anything in this pr it was working yesterday, fuck off
